android/src/main/java/com/crazecoder/openfile/OpenFilePlugin.java

```diff
@@ -151,9 +151,19 @@ private boolean pathRequiresPermission() {
         }
 
         try {
-            String appDirCanonicalPath = new File(context.getApplicationInfo().dataDir).getCanonicalPath();
```

Comment:
Why did you delete this line?
The app does not need permission to access its own data directory.